validator/src/main.rs

```diff
@@ -774,6 +774,12 @@ pub fn main() {
                 .takes_value(true)
                 .help("Where to store the tower"),
         )
+        .arg(
+            Arg::with_name("tower_migration")
+                .long("tower-migration")
+                .takes_value(false)
+                .help("Only use this flag when migrating from the old tower in version <= 1_7_14. Improper use of this flagcould result in a corrupted tower")
```

Comment:
nit: flagcould -> flag could

Reply:
will validators have to manually run this migration?

Reply:
yes when restarting with this change validators will need to pass this flag in order to deserialize their saved tower into the new version. Otherwise they'll still have the old version. Originally I was just going to deserialize the saved tower as both old/new and see which one worked in order to distinguish the version, however there is a possibility that a new tower could deserialize as a corrupted old tower.

Reply:
So i thought it would be best to have a manual flag - what do you think?

Reply:
hmmm, I think getting people to coordinate this in communication is going to be a PITA, people suck at following directions.

I think it's probably easier to use something like a separate naming scheme for one type of tower vs the other, and run the conversion if the tower name is missing.

Reply:
Seems like we can just modify the filename to include a version?